src/index.html

```diff
@@ -10,8 +10,176 @@
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
+
+
   <body>
-    <h1>HTML Form</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+    >
+      <fieldset class="part">
+        <legend>Personal information</legend>
+
+        <label class="form-field">
+          Surname:
+          <input
+            type="text"
+            name="Surname:"
```

Comment:
[checklist](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md) 4th item:
[CODE STYLE] - Use camelCase for values of name attribute - they should be valid as JavaScript object keys. It should not contain spaces, “-” or other special characters.
```suggestion
            name="surname"
```
make sure to fix it everywhere